Dockerfile.kernel

```diff
@@ -1,25 +1,18 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-#FROM jupyter/minimal-kernel
 FROM jupyter/minimal-kernel:9cb6443840b2
 
 USER root
 
 # install latest version of node & npm (this and git clone also requires ca-certs)
-RUN apt-get install -yq --no-install-recommends ca-certificates curl
+RUN apt-get update && \
+    apt-get install -yq --no-install-recommends ca-certificates curl
 RUN curl -sL https://deb.nodesource.com/setup_5.x | bash -
+RUN apt-get update && \
```

Comment:
Docker best practice: the prior line might be cached from weeks ago, but you changed the second line. Ubuntu / Debian like to rev packages with security fixes by taking them off the inet. When that happens, and you haven't run apt-get update again, you fail.